LoginPlugin/iOS/README.md

```diff
@@ -0,0 +1,147 @@
+# ZappLoginPluginExample-iOS
+
+The ZappLoginPluginExample is an example project for creating a login plugin for the Applicaster Zapp Platform. You can use this example project as a reference for how to build your own login plugin.
+
+If you are not familiar with Zapp please visit [our website](http://applicaster.com/?page=product) for more details.
+
+The full [Zapp](http://zapp.applicaster.com) plugins documentation is available [here](http://zapp-tech-book.herokuapp.com).
```

Comment:
please change 'http://zapp-tech-book.herokuapp.com' to 'http://developer-zapp.applicaster.com/'